pkg/cond/parser_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 Chainguard, Inc.
+//
```

Comment:
can we add a simple test for broken conditional? something like `foo ==` (missing second half)? It looks like this might cause a panic

Reply:
Sure, but panics are caught inside `goparsify`.